### PR TITLE
Add lister component

### LINKED ISSUE #32: Port lister component

### PR DESCRIPTION
**Pull request by anita-chouhan**
Fixes #32

### CODE REVIEW

File: templates/_components/_lister.twig [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  @param  class            {string}   Class names for list wrapper element
+  @param  list_class       {string}   Class names for <ul> tag element
+  @param  list_attributes  {array}    Attributes for <ul> tag element
+  @param  item_class       {string}   Class names for <li> tag element
+
+
+
+  Configuration
+  =============
+  There is a configurable option available which help save
```
Comment (rungta) [inline templates/_components/_lister.twig]: ... _which helps avoid some repetition_ ...

File: templates/_components/_lister.twig [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+
+
+{# Config #}
+{% set listerProfiles = {} %}
+
+
+
+{# Variables #}
+{% set listerConfig = listerProfiles[listerKind] ?? listerProfiles.teaser %}
```
Comment (rungta) [inline templates/_components/_lister.twig]: `listerProfiles.teaser` may not be defined (right now it is an empty array) so we should have a fallback `?? {}`.

File: templates/_components/_lister.twig [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+  ListerProfiles
+  ----------
+  A set of pre-configured values for list generation. Define the desired
+  configurations in the `listerProfiles` array using the following options:
+  
+  @option  view  {string}  Name of view type that you want to use for each list item eg: 'teaser' | 'title' etc.
+  @option  list  {string}  Class names for <ul> tag element
+  @option  item  {string}  Class names for <li> tag element
+
```
Comment (rungta) [inline templates/_components/_lister.twig:33]: A `wrap` option is also supported which will add a class to a wrapper around the list. (See line 88)

Comment (rungta) [inline templates/_components/_lister.twig:33]: A `itemWrap` option is also supported. (See line 92)

**Review by rungta — CHANGES_REQUESTED** [review]

File: templates/_components/_lister.twig (hunk 1 above)
Comment (anita-chouhan) [reply templates/_components/_lister.twig]: Done.

File: templates/_components/_lister.twig (hunk 2 above)
Comment (anita-chouhan) [reply templates/_components/_lister.twig]: Done.

File: templates/_components/_lister.twig (hunk 3 above)
Comment (anita-chouhan) [reply templates/_components/_lister.twig:33]: Added to comment section.

Comment (anita-chouhan) [reply templates/_components/_lister.twig:33]: Added to comment section.

Comment (rungta) [reply templates/_components/_lister.twig:33]: But which section did you add it to? Please read the code and figure out _where_ this parameter is supported and hence where should it be documented.

Comment (rungta) [reply templates/_components/_lister.twig:33]: Again, which section did you add it to? Same response as for the `wrap` option.

File: templates/_components/_lister.twig (hunk 2 above)
Comment (rungta) [reply templates/_components/_lister.twig]: The `( )` brackets are redundant.

File: templates/_components/_lister.twig (hunk 3 above)
Comment (anita-chouhan) [reply templates/_components/_lister.twig:33]: Sorry, moved the comments to the right section.

Comment (anita-chouhan) [reply templates/_components/_lister.twig:33]: Updated.

**Review by rungta — APPROVED** [review]

**Pull request merged**